lua/luasnip/util/util.lua

```diff
@@ -233,12 +233,22 @@ local function put(text, pos)
 	pos[2] = (#text > 1 and 0 or pos[2]) + #text[#text]
 end
 
--- Wrap a value in a table if it isn't one already
+--[[ Wraps the value in a table if it's not one, makes
+  the first element an empty str if the table is empty]]
 local function wrap_value(value)
```

Comment:
How about renaming this function to just `wrap_string` or `to_string_table` (I think it's only used for that right now either way, but better to be specific)?